docs/proxy/google_serp_proxy/nodejs_examples.md

```diff
@@ -27,9 +31,9 @@ Get a list of results from the US for keyword `wikipedia` and parse them through
 
         const crawler = new Apify.PuppeteerCrawler({
             requestList,
+            proxyConfiguration,
             launchPuppeteerOptions: {
-                headless: true,
-                proxyUrl,
+                headless: true
             },
```

Comment:
I'd get rid of the `headless` and consequently the whole `launchPuppeteerOptions`. It's managed automatically by the SDK based on where the script is running.

Reply:
Sure, removed.